src/gui/MidiCCRackView.cpp

```diff
@@ -49,7 +49,7 @@ MidiCCRackView::MidiCCRackView(InstrumentTrack * track) :
 	setWindowIcon(embed::getIconPixmap("midi_cc_rack"));
 	setWindowTitle(tr("MIDI CC Rack - %1").arg(m_track->name()));
 
-	QMdiSubWindow * subWin = getGUI()->mainWindow()->addWindowedWidget(this);
+	SubWindow * subWin = getGUI()->mainWindow()->addWindowedWidget(this);
```

Comment:
```suggestion
	SubWindow* subWin = getGUI()->mainWindow()->addWindowedWidget(this);
```
The asterisk should go on the left for new code